config/resolvers/resolvers-deployment.yaml

```diff
@@ -89,7 +89,7 @@ spec:
         - name: CONFIG_FEATURE_FLAGS_NAME
           value: feature-flags
         - name: CONFIG_LEADERELECTION_NAME
-          value: config-leader-election
+          value: config-leader-election-resolvers
```

Comment:
Should I also change for resolvers? I did in case someone try to install in same namespace as pipelines controllers.

Reply:
I think it make sense yes